scripts/testnet/batchSettleEcoTestNet.ts

```diff
@@ -518,26 +540,26 @@ async function proveWorldStateBaseSepoliaOnOptimismSepolia(
   }
 }
 
-async function proveWorldStateBedrockOnOptimismSepoliaforEcoTestNet(
+async function proveWorldStateBedrockOnOptimismSepoliaforEcoTestnet(
```

Comment:
maybe a nit, maybe there is some objectivity to this, i think we gotta find a way to make these method names and variable names shorter, for readability and also mistake prevention.